python/cudf/cudf/pylibcudf_tests/test_datetime.py

```diff
@@ -0,0 +1,35 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.
+
+import datetime
+
+import pyarrow as pa
+import pytest
+from utils import assert_column_eq
+
+import cudf._lib.pylibcudf as plc
+
+
+@pytest.fixture(params=[False, True])
+def nullable(request):
+    return request.param
```

Comment:
This fixture must be propagating. Should we move it to conftest.py? It's so basic and omnipresent.

Reply:
Can do. Although in doing so, let's paint the bikeshed, I think I prefer `has_nulls` over `nullable` (despite having written nullable here!), because I've don't particularly like `nullable` as an adjectival modifier of columns: to me "this is a nullable column" means "it could have nulls" rather than "it does have nulls".

Reply:
I agree. It's also worth noting the distinction in libcudf: `has_nulls` means that a column has nulls, while `nullable` just means that it has an allocated (but potentially all valid) null mask. We really only care to test the former.

Reply:
Done.